### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by kenhitchcock**

### CODE REVIEW

**Comment by danhawker** [comment]
@kenhitchcock Sorry, not been paying attention lately, so missed this. Looks like mostly libvirt stuff is that fair?

File: tasks/vm-aws.yml [hunk 1]
```diff
@@ -96,6 +102,7 @@
       Project: "{{ cornerstone_prefix }}"
       Purpose: "{{ cornerstone_tag_purpose }}"
       Role: "{{ cornerstone_tag_role }}"
+      kubernetes.io/cluster/openshift: "clusterid"
```
Comment (danhawker) [inline tasks/vm-aws.yml]: Slightly confused why this is here? We probably ought to come up with a better way of adding tags generally.

**Comment by kenhitchcock** [comment]
This was me testing when i was deploying openshift. This tag was needed and
i short cutted it :( It really should be removed and not have been
committed to the code. Will fix and push again.

**Comment by danhawker** [comment]
Ah, makes sense. Happy to commit for rest

**Comment by danhawker** [comment]
LGTM

**Pull request merged**